pages/profile.js

```diff
@@ -0,0 +1,12 @@
+import Layout from '../components/Layout';
+import { useState } from 'react';
+
+const [ profile, setProfile ] = useState({ email: ''});
+
+export default function Profile() {
```

Comment:
close, but you want to export default function `withAuthSync(Profile)`
if the HOC works as it should

Reply:
ah ok, thanks a lot, Jesus!